scripts/closure-compiler/build-devapp-bundle.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Script that bundles the dev-app using the Google Closure compiler.
+# This is script is used to verify closure-compatiblity of all Material components.
```

Comment:
This PR just bundles our demo-app that uses Angular Material. So not really sure if this is enough for the variable renaming. Can be a follow-up?